_includes/header-technical.html

```diff
@@ -1,3 +1,3 @@
 <header id="top" class="header-technical container-fluid">
-  <a class="header-logo" href="{{site.baseurl}}"><img src="../img/logo-firefox-os-phoxygen.png"></a>
+  <a class="header-logo" href="{{site.baseurl}}/"><img src="../img/logo-firefox-os-phoxygen.png"></a>
```

Comment:
I would rather change the config to add the / to site.baseurl. That's what I have locally (didn't have a chance to submit that yet). Alternatively, we can define a `homeurl` variable in the config.